src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -25,6 +25,7 @@
  */
 package jenkins.security.plugins.ldap;
 
+import com.trilead.ssh2.crypto.Base64;
```

Comment:
use the one from commons-codec, don't rely on trilead as the plan will be to strip that from core eventually